assets/scss/_submission.scss

```diff
@@ -113,8 +113,7 @@
             text-align: center;
           }
 
-          &:nth-child(4),
-          &:last-child {
+          &:nth-child(4) {
```

Comment:
Can you make the last child (last column) also right aligned please?

Reply:
the status you mean? it is not really pretty when right aligned and the status was centered before.

Reply:
the status you mean? it is not really pretty when right aligned and the status was centered before.

Reply:
Yes, but the status wasn't the last column 😉 But ok then, it's a matter of taste.

Reply:
I see. Let's try something.

Reply:
how about this?

![screen shot 2018-01-30 at 16 21 03](https://user-images.githubusercontent.com/217628/35574269-a00e4156-05d9-11e8-8de9-0ca3a39abafb.png)

Reply:
👍